python/tests/harvest_pubmed.py

```diff
@@ -27,7 +27,7 @@ def test_pubmed_harvest_date(mocker):
 
     # We'll need one entry in the date_file_map.
     generate_date_file_map = mocker.patch("fatcat_tools.harvest.pubmed.generate_date_file_map")
-    generate_date_file_map.return_value = {test_date: set(["dummy"])}
+    generate_date_file_map.return_value = {test_date: {"dummy"}}
```

Comment:
This is a particularly strong example of confusing syntax when using the curly-bracket syntax for set construction (in my subjective opinion). AKA, I think the explicit `set()` is better here.